suave/backends/redis_store_backend.go

```diff
@@ -39,6 +45,16 @@ func NewRedisStoreBackend(redisUri string) *RedisStoreBackend {
 }
 
 func (r *RedisStoreBackend) Start() error {
+	if r.redisUri == "" {
+		// create a mini-redis instance
```

Comment:
It defaults to local `miniredis` if no endpoint is specified. Then, the rest of the store works for both remote/local Redis.